stratum/hal/lib/p4/utils.cc

```diff
@@ -117,6 +117,32 @@ std::string ByteStringToP4RuntimeByteString(std::string bytes) {
   return bytes;
 }
 
+::util::Status IsValidMeterConfig(const ::p4::v1::MeterConfig& meter_config) {
+  if (meter_config.cir() > meter_config.pir()) {
+    RETURN_ERROR(ERR_INVALID_PARAM)
+        << "Meter configuration " << meter_config.ShortDebugString()
+        << " is invalid: committed rate cannot be greater than peak rate.";
+  }
+  if (meter_config.cburst() > meter_config.pburst()) {
+    RETURN_ERROR(ERR_INVALID_PARAM)
+        << "Meter configuration " << meter_config.ShortDebugString()
+        << " is invalid: committed burst size cannot be greater than peak burst"
+        << " size.";
+  }
```

Comment:
This check does not seem to be an actual requirement from the RFC. Maybe remove?